code-challenges.test.js

```diff
@@ -15,36 +15,77 @@
 
 // a) Create a test with an expect statement using the variable provided. 
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
+ describe('cutAndShuffle',() => {
+     const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+    const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+    it('removes the first item from the array and shuffles the rest.', () => {
+
+expect(cutAndShuffle(colors1)).toEqual(expect.arrayContaining(["yellow", "blue", "pink", "green"]))
+expect(cutAndShuffle(colors2)).toEqual(expect.arrayContaining(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]))
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+});
+})
 
 // b) Create the function that makes the test pass.
-
+// cut and shuffle intakes an array slices index one off from the array 
+// it will return new array and use sort with math random to shuffle the array each time the function runs
+const cutAndShuffle = (array) => {
+   let newArray = array.slice(1) 
+   return newArray.sort((a,b) => 0.5 - Math.random()) 
+}
```

Comment:
This is really great work Athian!  I really like that you are utilizing the sort method here!